features/step_definitions/kata.rb

```diff
@@ -117,7 +117,9 @@
   kata_ns ||= "openshift-sandboxed-containers-operator"
   kata_config_name = "example-kataconfig"
   step %Q/I store master major version in the :master_version clipboard/
-  raise "Kata operator OLM installation only supported for OCP >= 4.8" unless cb.master_version >= "4.8"
+  major = cb.master_version.split('.').first
+  minor = cb.master_version.split('.').last 
+  raise "Kata operator OLM installation only supported for OCP >= 4.8" unless major.to_i > 4 || (major.to_i == 4 && minor.to_i >= 8)
```

Comment:
Do we have major > 4 ? Perhaps we can simplify it to `unless minor_version.to_i >= 8` for now

Reply:
If we needn't consider OCP 5.x in advance, then we could use "unless minor_version.to_i >= 8" for now